Framework/Algorithms/src/ChangePulsetime2.cpp

```diff
@@ -7,8 +7,8 @@
 namespace Mantid {
 namespace Algorithms {
 
-// Register the algorithm into the AlgorithmFactory
-DECLARE_ALGORITHM(ChangePulsetime2)
+// Algorithm not registered as this will be used for testing MPI
+// DECLARE_ALGORITHM(ChangePulsetime2)
```

Comment:
This can be uncommented to test the ChangePulsetime2 algorithm with the GUI (workspace history, dialog box generation etc.)